### PR TITLE
Remove deprecated LibrarySort

### PR DESCRIPTION
**Pull request by ghostbear**

### CODE REVIEW

**Review by 53jk1 — APPROVED** [review]

File: app/src/main/java/eu/kanade/tachiyomi/Migrations.kt [hunk 1]
```diff
@@ -105,9 +104,9 @@ object Migrations {
                 val oldSortingMode = prefs.getInt(PreferenceKeys.librarySortingMode, 0)
 
                 @Suppress("DEPRECATION")
```
Comment (stevenyomi) [inline app/src/main/java/eu/kanade/tachiyomi/Migrations.kt:106]: I guess this `@Suppress` can be removed? Also the one in line 203.

**Review by arkon — COMMENTED** [review]
It'd be worth just leaving comments on what the numbers mean for better context.

File: app/src/main/java/eu/kanade/tachiyomi/Migrations.kt [hunk 2]
```diff
@@ -104,10 +103,9 @@ object Migrations {
                 // Reset sorting preference if using removed sort by source
                 val oldSortingMode = prefs.getInt(PreferenceKeys.librarySortingMode, 0)
 
-                @Suppress("DEPRECATION")
-                if (oldSortingMode == LibrarySort.SOURCE) {
+                if (oldSortingMode == 5) {
```
Comment (ghostbear) [inline app/src/main/java/eu/kanade/tachiyomi/Migrations.kt:106]:
```suggestion
                if (oldSortingMode == 5 /* SOURCE */) {
```

File: app/src/main/java/eu/kanade/tachiyomi/Migrations.kt [hunk 3]
```diff
@@ -104,10 +103,9 @@ object Migrations {
...
                     prefs.edit {
-                        putInt(PreferenceKeys.librarySortingMode, LibrarySort.ALPHA)
+                        putInt(PreferenceKeys.librarySortingMode, 0)
```
Comment (ghostbear) [inline app/src/main/java/eu/kanade/tachiyomi/Migrations.kt:108]:
```suggestion
                        putInt(PreferenceKeys.librarySortingMode, 0 /* ALPHABETICAL */)
```

**Review by ghostbear — COMMENTED** [review]
I will leave you the `when` for comments as what on the right is more or less what the number mean

**Pull request merged**